package.json

```diff
@@ -21,10 +21,12 @@
     "@vue/cli-plugin-router": "~5.0.0",
     "@vue/cli-plugin-vuex": "~5.0.0",
     "@vue/cli-service": "~5.0.0",
-    "eslint": "^7.32.0",
+    "eslint": "^8.24.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-vue": "^8.0.3",
+    "eslint-plugin-pug": "^1.2.4",
+    "eslint-plugin-vue": "^9.6.0",
+    "eslint-plugin-vue-pug": "^0.5.4",
     "prettier": "^2.4.1",
```

Comment:
you can add `"@prettier/plugin-pug"` to fix whitespaces but, currently there'll no error in the IDE but it'll fix through CLI. Hence, it can be prettified with lint-staged

Reply:
Okay. Thanks for your suggestions. 

Firstly, I added `"@prettier/plugin-pug"` plugin to the project,
Secondly, I set the lint-staged on the package.json file.

I did nothing about pre-commit with husky for now.